### PR TITLE
mailing-lists.json: avoid getting objects

### PR DESCRIPTION
**Pull request by reinhardt**
syslabcom/scrum#1941

### CODE REVIEW

File: src/osha/oira/client/browser/client.py [hunk 1]
```diff
@@ -182,30 +183,7 @@ def results(self):
 
             brains = catalog(**query)
 
-            sectors = api.portal.get().sectors
-
-            def filter_items(brain):
-                obj = brain.getObject()
-
-                if getattr(obj, "preview", False) or getattr(obj, "obsolete", False):
```
Comment (reinhardt) [inline src/osha/oira/client/browser/client.py:190]: These two lines are not ported. A user can subscribe to a preview and an obsolete tool, so it doesn't make sense to exclude them here. And even if it did, the check would have to be made on the counterpart, not the client object.

File: src/osha/oira/client/indexers.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+def is_in_client(obj):
+    for parent in obj.aq_chain:
+        if IClient.providedBy(parent):
+            return True
+    return False
+
+
+@indexer(Interface)
+def managerRolesAndUsers(client_obj):
```
Comment (thet) [inline src/osha/oira/client/indexers.py:18]: A‌ little explanation on what this index does would be nice.

**Review by thet — APPROVED** [review]
A bit more inline comments would be helpful - speaking from experience with cyberhead junior where no such comments exist.
However, apart from that LGTM and mergeable right away.

File: src/osha/oira/client/indexers.py (hunk 2 above)
Comment (thet) [reply src/osha/oira/client/indexers.py:18]: Super!

**Review by thet — APPROVED** [review]

**Pull request merged**